File: code_samples/java/NetworkingSocketProgramming.java
/**
 * Networking: Socket Programming
 * AI/ML Training Sample
 */
public class NetworkingSocketProgramming {
    
    private String data;
    
    public NetworkingSocketProgramming() {
        this.data = "";
    }
    
    public void process(String input) {
        this.data = input;
    }
    
    public String getData() {
        return this.data;
    }
    
    public boolean validate() {
        return this.data != null && !this.data.isEmpty();
    }
    
    public static void main(String[] args) {
        NetworkingSocketProgramming instance = new NetworkingSocketProgramming();
        instance.process("example");
        System.out.println("Data: " + instance.getData());
        System.out.println("Valid: " + instance.validate());
    }
}

File: code_samples/csharp/networking_socket_programming.cs
using System;

/// <summary>
/// Networking: Socket Programming
/// AI/ML Training Sample
/// </summary>
public class SocketProgramming
{
    private string data;
    
    public SocketProgramming()
    {
        this.data = string.Empty;
    }
    
    public void Process(string input)
    {
        this.data = input;
    }
    
    public string GetData()
    {
        return this.data;
    }
    
    public bool Validate()
    {
        return !string.IsNullOrEmpty(this.data);
    }
    
    public static void Main(string[] args)
    {
        var instance = new SocketProgramming();
        instance.Process("example");
        Console.WriteLine($"Data: {instance.GetData()}");
        Console.WriteLine($"Valid: {instance.Validate()}");
    }
}
